packages/sdk/src/methods/add-to-list/AddToList.ts

```diff
@@ -31,10 +32,13 @@ export function addCSDGroupToList(session: AbstractSession, parms: ICSDGroupParm
 
   Logger.getAppLogger().debug("Attempting to add a CSD Group to a CSD List with the following parameters:\n%s", JSON.stringify(parms));
 
-  const cicsPlex = parms.cicsPlex == null ? "" : parms.cicsPlex + "/";
-  const cmciResource = "/" + CicsCmciConstants.CICS_SYSTEM_MANAGEMENT + "/" +
-        CicsCmciConstants.CICS_CSDGROUP + "/" + cicsPlex + parms.regionName +
-        "?CRITERIA=NAME=='" + parms.name + "'";
+  const options: IGetResourceUriOptions = {
+    "cicsPlex": parms.cicsPlex,
+    "regionName": parms.regionName,
+    "criteria": `NAME=='${parms.name}'`
```

Comment:
Not a required change, but as a style recommendation (and to save a few extra key strokes... you can avoid the quotes in objects (as long as there are no special characters in the property/key name 😋 
```suggestion
    cicsPlex: parms.cicsPlex,
    regionName: parms.regionName,
    criteria: `NAME=='${parms.name}'`
```